packages/bot-handler-wizard/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "dependencies": {
+    "wizardy": "0.2.6"
+  },
+  "devDependencies": {
+    "@types/jasmine": "3.5.10",
+    "@types/node": "~12",
+    "@wireapp/bot-api": "7.10.27",
```

Comment:
I think this should be in the dependencies.

Reply:
No. Bot API is an optional dependency. It's just listed in dev dependencies so that tests can work with it. It's the pattern we use for all other bot handlers.